src/app/containers/PageHandlers/withData/shouldRender/index.test.js

```diff
@@ -0,0 +1,99 @@
+import shouldRender from '.';
```

Comment:
I'm not convinced by the tests here. `shouldRender` looks like a pure function with no side effects however not one test checks the actual output of the function. Instead it mocks utility functions and makes sure they are called. If someone was to refactor `shouldRender`, as long as these utility functions are always called then the tests will pass even if they accidentally changed the output of the function.